app/views/distributions/pick_ups.html.erb

```diff
@@ -4,9 +4,9 @@
     <div class="container-fluid">
       <div class="row mb-2">
         <div class="col-sm-7">
-          <% content_for :title, "Distribution Pick-Ups - #{current_organization.name}" %>
+          <% content_for :title, "Distribution Schedule - #{current_organization.name}" %>
```

Comment:
We should probably rename this file too, but that can be a separate PR

Reply:
@albertchae sounds good. Could either of you create the issue to handle that and label it as `tech-debt` after this has been merged in?

Reply:
Finally got around to creating this issue. https://github.com/rubyforgood/diaper/issues/1942